Revit_oM/Config/PullGeometryConfig.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Adapter;
+using System.ComponentModel;
+
+namespace BH.oM.Adapters.Revit
+{
+    [Description("Configuration used to specify which geometry should be pulled and passed to CustomData.")]
+    public class PullGeometryConfig
+    {
+        /***************************************************/
+        /****             Public Properties             ****/
+        /***************************************************/
+
+        [Description("If true, edges of elements will be pulled and stored under Revit_edges in CustomData.")]
+        public virtual bool PullEdges { get; set; } = false;
+
+        [Description("If true, surfaces of elements will be pulled and stored under Revit_surfaces in CustomData.")]
+        public virtual bool PullSurfaces { get; set; } = false;
+
```

Comment:
Can we add PullMeshes here? I appreciate you can pull through Representation, but I think intuitive to have on the geometry config also. We should be able to deal with only pulling once then, on the RevitPullConfig.

![image](https://user-images.githubusercontent.com/15233608/81000436-2fd7a680-8e3e-11ea-83ee-5141da96c8bf.png)

Reply:
Not sure about this one, I see 3 issues with that:
1. This would be duplication  of pull or representation
2. As discussed, geometry links to engineering, while representation to preview - I like this separation and would prefer not to blur it
3. Detail level of geometry pull is defaulted to _Fine_, while for representation it is a variable. Both options of adding a detail level property to `PullGeometryConfig` or pulling fine meshes by default (slooow) do not sound good to me.